### ISSUE TITLE
Regarding fees

### ISSUE THREAD

**Issue by kingkhanraza** [matter-labs/era-contracts#205]
#### some dapp are charging much fess
Cant judge how fess will be return
Pls fix this

**Comment by kingkhanraza** [comment]
Some dex for same swap .10 cent some 1 doller
Example syncswap lower fess and izumi pancakeswap charging 1 doller

**Comment by amazonjapan** [comment]
The gas fees influenced by the state-diff-based data availability and the costs of publishing the contract code

**Comment by maskatay** [comment]
> Some dex for same swap .10 cent some 1 doller Example syncswap lower fess and izumi pancakeswap charging 1 doller

So true but many factors such as availability influences fees.

**Comment by kingkhanraza** [comment]
But this is impacting user 
Also i am hisitating explore more dapp on zk 
Because unstable fess on every protocal
Its 3x to 10× diffrence

**Comment by Abkeys1113** [comment]
Yes  I was sending a token recently and it cost me 3 bucks in fee
I was making transactions on even syncswap it showed few cents in fee but onchain it was  dollar nd 50cents

**Comment by Abkeys1113** [comment]
Overall tho zk is my favorite mhen I love love trading nfts here I'll build my casino here 🫶🏽
Sending love from Africa❤️
Good job team
